crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -156,6 +116,110 @@ pub async fn create(
     })
 }
 
+pub async fn import_from_keystore(
+    provider: &JsonRpcClient<HttpTransport>,
+    keystore_path: Utf8PathBuf,
+    account_path: Utf8PathBuf,
+) -> Result<(serde_json::Value, u64)> {
+    let account_info: serde_json::Value = serde_json::from_str(&fs::read_to_string(account_path)?)?;
+    let deployment = account_info
+        .get("deployment")
+        .ok_or_else(|| anyhow!("No deployment field in account JSON file"))?;
+
+    let deployed = match deployment.get("status").and_then(serde_json::Value::as_str) {
+        Some("deployed") => true,
+        Some("undeployed") => false,
+        _ => bail!("Unknown deployment status value"),
+    };
+
+    let salt: Option<FieldElement> = {
+        let salt = deployment.get("salt").and_then(serde_json::Value::as_str);
+        match (salt, deployed) {
+            (Some(salt), _) => Some(parse_number(salt)?),
+            (None, false) => bail!("No salt field in account JSON file"),
+            (None, true) => None,
+        }
+    };
+
+    let class_hash: FieldElement = {
+        let ch = deployment
+            .get("class_hash")
+            .and_then(serde_json::Value::as_str)
+            .ok_or_else(|| anyhow!("No class_hash field in account JSON file"))?;
```

Comment:
are those accounts by default OZ? if yes, we could just use their class hash, and not fail.

Reply:
They are 👀

Reply:
Thought about it and I think explicit error is better here.
The starkli account json is something that shouldn't be modified manually, hence it should be of the desired schema (and have the class hash among other fields).